roles/core/access_control/tasks/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: include_tasks ░ Use OS dedicated task
+  include_tasks: "{{ (ansible_facts.distribution|lower|replace(' ','_')) }}_{{ ansible_facts.distribution_major_version }}.yml"
```

Comment:
You can simplify with `ansible_facts.os_family` since centos_8.yml and redhat_8.yml are identical.

Reply:
Yes. Deleted centos 8 in my local branch.

Reply:
And updated task with ansible_facts.os_family